core/src/main/scala/org/apache/spark/sql/execution/columnar/impl/ColumnFormatRelation.scala

```diff
@@ -80,11 +81,13 @@ class BaseColumnFormatRelation(
     with PartitionedDataSourceScan
     with RowInsertableRelation {
 
+  val scanAsUnsafeRows = false
```

Comment:
Actually I plan to drop this completely once I get precheckin fully clean and one set of TPCH run completed (hopefully by tomorrow). Hence not added SQLConf setting since it is very temporary mostly for my local testing (for TPCH we already have previous results to compare against).